### PR TITLE
Add MAPE objective

### PR DESCRIPTION
**Pull request by Reddine**

### CODE REVIEW

File: keras/objectives.py [hunk 1]
```diff
@@ -12,6 +12,9 @@ def mean_squared_error(y_true, y_pred):
 def mean_absolute_error(y_true, y_pred):
     return T.abs_(y_pred - y_true).mean(axis=-1)
 
+def mean_absolute_percentage_error(y_true, y_pred):
+    return T.abs_((y_true - y_pred) / y_true).mean() * 100
```
Comment (fchollet) [inline keras/objectives.py]: Since you are taking a global mean instead of the mean on the last axis, this will fail for loss weighting.

Comment (Reddine) [inline keras/objectives.py]: fixed :+1:

**Comment by fchollet** [comment]
If also very common for labels to be be 0. In this case 0 labels would trigger a NaN.

**Comment by fchollet** [comment]
Fixed and merged.

**Comment by Reddine** [comment]
this objective is useful in case of learning an approximation of a function as in my current project : load forecating

**Pull request merged**